tests/fixture/tmpnet/network.go

```diff
@@ -1017,12 +1018,23 @@ type RPCChainVMVersion struct {
 
 // getRPCVersion attempts to invoke the given command with the specified version arguments and
 // retrieve an rpcchainvm version from its output.
-func getRPCVersion(command string, versionArgs ...string) (uint64, error) {
+func getRPCVersion(log logging.Logger, command string, versionArgs ...string) (uint64, error) {
 	cmd := exec.Command(command, versionArgs...)
 	output, err := cmd.CombinedOutput()
 	if err != nil {
 		return 0, fmt.Errorf("command %q failed with output: %s", command, output)
 	}
+
+	// Ignore output before the opening brace to tolerate the case of a command being invoked
+	// with `go run` and the go toolchain emitting diagnostic logging before the version output.
+	if idx := bytes.IndexByte(output, '{'); idx != -1 {
+		log.Info("ignoring leading bytes of JSON version output in advance of opening `{`",
+			zap.String("command", command),
+			zap.String("ignoredLeadingBytes", string(output[:idx])),
+		)
```

Comment:
I think this will log even if `idx == 0` (aka there are no leadingBytes). Is that expected? Perhaps the if should be `idx > 0` rather than `idx != -1`